src/locales/en.json5

```diff
@@ -1,10 +1,43 @@
 // action: This is the text for an action, for example, the text of a button.
 // field: This is the text of a form field.
 // header: This is the text of a table column header.
-
+// title: This is text of the document/page title shown in the browser tab and in browser history.
```

Comment:
This comment looks great — I think it'll be helpful to translators. You can actually add it right above `"title": {` below. Doing so will make sure the comment is visible in Transifex for each message in the `title` object. Defining the comment at the top of `en.json5` would apply it even more broadly: not just to this `title` message, but to any message with a `title` key, including in components. (And actually, if you look at the autogenerated `strings_en.json` below, you can see that a couple of unrelated messages received the comment.)